### PR TITLE
Keep branch 1.x on the old API, use 2.x for the new API

### PR DESCRIPTION
**Pull request by monkbroc**
This is my suggestion for #125: Keep 1.x with the old API by reverting 1.4 to 1.6 and use 2.x for the new API by applying to the changes of 1.5 to 2.0.

### CODE REVIEW

**Comment by Jaco-Pretorius** [comment]
:+1:

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,25 @@
...
+This is the last version that uses the API V1.
+
+Since the migration from V1 to V2 has no breaking changes probably you are good to upgrade to 2.x. Otherwise lock the gem to 1.x in your `Gemfile`: `gem "filepicker-rails", "~> 1.0"`
+
+More info about the migration [here](https://www.filepicker.com/documentation/file_ingestion/javascript_api/migration?v=v2).
+
+### features
+
+- The latest gem 1.x stays with API V1
+- Revert the changes of 1.5.0 and 1.5.1 and re-release 1.4.0 as 1.6.0
```
Comment (maurogeorge) [inline CHANGELOG.md]: We do not need to re-release the 1.4.0 as 1.6.0 we can simply [yank the 1.5.0 and 1.5.1](http://help.rubygems.org/kb/gemcutter/removing-a-published-rubygem).

Comment (maurogeorge) [inline CHANGELOG.md]: This way the last release will be the 1.4.0 as you can see [here](http://help.rubygems.org/kb/gemcutter/removing-a-published-rubygem)

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,25 @@
 # Changelog
 
+## 2.0.0 (July, 15, 2015)
+
+### features
+
+- Use API V2
+- Re-release the changes of 1.5.0 and 1.5.1 as 2.0.0
```
Comment (maurogeorge) [inline CHANGELOG.md]: This was done as you can see here https://github.com/Ink/filepicker-rails/blob/master/CHANGELOG.md

**Comment by maurogeorge** [comment]
The 2.0.0 was released as you can see here https://github.com/Ink/filepicker-rails/issues/125#issuecomment-121765461

If it is ok for you please close this.

**Comment by monkbroc** [comment]
In cases other than security vulnerabilities, I prefer not to yank gem version since it would prevent people using those version from deploying their code, messing up their CI, etc. That's why I suggested simply re-releasing 1.4 as 1.6 then re-releasing 1.5 as 2.0.

In any case, thanks for releasing 2.0.

**Comment by maurogeorge** [comment]
@monkbroc I decided to yanked based [on](https://github.com/thoughtbot/shoulda-matchers/blob/master/NEWS.md#v-153---yanked-due-to-mis-release) [other](https://github.com/rspec/rspec-rails/blob/master/Changelog.md#2114--2012-10-14) [projects](https://github.com/plataformatec/devise/blob/master/CHANGELOG.md#351---2015-05-24). The 1.6 makes sense too

**Pull request closed without merging**